src/status_im/constants.cljs

```diff
@@ -431,6 +431,8 @@
 (def ^:const optimism-network-name :optimism)
 (def ^:const arbitrum-network-name :arbitrum)
 
+(def ^:const default-network-names #{:ethereum :optimism :arbitrum})
```

Comment:
We have these keywords defined already as `mainnet-network-name`, `opt..-netw..-name` etc.

It's not a huge deal but we should try to not introduce duplication in config. 
```clj
(def ^:const default-network-names #{mainnet-network-name optimism-network-name arbitrum-network-name})
```
is better

Reply:
Agreed and updated 👍  (Rookie mistake 😅)

Reply:
No, it's quite easy to overlook such a tiny detail.